demos/utils/utils.ts

```diff
@@ -0,0 +1,103 @@
+import fetch from "node-fetch";
+import fs from "fs-extra";
+import { promisify } from "util";
+import { Stream } from "stream";
+import { ChildProcess, spawn as nodeSpawn } from "child_process";
+import { cwd } from "process";
+import path from "path";
+
+const pipeline = promisify(Stream.pipeline);
+const child_processes: ChildProcess[] = [];
+
+process.on("exit", () => {
+  killAllChildren();
```

Comment:
This was a hack to ensure avoid problems during workshops, I would prefer something less hacky for demos if possible